pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerDe.scala

```diff
@@ -282,3 +282,9 @@ object SerDe {
     value.foreach(v => writeBytes(out, v))
   }
 }
+
+object SerializationFormats {
```

Comment:
That's a good point; the easiest way I can think of is one additional implicit on Scala side, which does sound like more code to maintain :)  Feel free to drop this one @cafreeman.